test/yaml_lock/run_test.py

```diff
@@ -2,3 +2,36 @@
 # call `cppdock init` in this working dir
 # test that yaml file is the same except for specifying revisions in each dep
 # The dep with the hardcoded revision should NOT change
+
+import os
+import subprocess
+import sys
+import yaml
+
+boosthana = 'd9da8776aa1142c0f92a38d49829ae5c3e86c7bc'
+
+def start():
+
+    p = subprocess.Popen(['cppdock', 'init'], stdin=subprocess.PIPE)
+    out, err = p.communicate('')
+    if p.returncode != 0:
+        print """
+        FAILURE
+        """
+
+    stream = file('cppdock2.yaml', 'r')
+    config = yaml.load(stream)
+
+    boost = config['platforms'][1]['deps'][1]['revision']
+    if not boost == boosthana:
+        raise ValueError('boosthana value changed')
```

Comment:
This error message isn't very informative. Try something like `Locked revision changed`